crates/swc_ecma_transforms_base/src/rename/eval.rs

```diff
@@ -31,9 +31,17 @@ impl Visit for EvalFinder {
 
     fn visit_export_namespace_specifier(&mut self, _: &ExportNamespaceSpecifier) {}
 
-    fn visit_ident(&mut self, i: &Ident) {
-        if i.sym == js_word!("eval") {
-            self.found = true;
+    fn visit_callee(&mut self, c: &Callee) {
+        c.visit_children_with(self);
+
+        if let Callee::Expr(e) = c {
+            if let Expr::Ident(Ident {
+                sym: js_word!("eval"),
```

Comment:
Not sure how you represent optional chains, but `eval?.()` is not a direct eval either.

Reply:
<img width="725" alt="image" src="https://user-images.githubusercontent.com/29931815/197099671-a07dd716-359c-4e12-9d59-e4835e3e63d7.png">

Nice catch, thank you!

Reply:
Oh it was `OptCall` which does not contain `Callee`